### ISSUE TITLE
【質問】発注時間と約定時間の差について

### ISSUE THREAD

**Issue by zmp365** [kabucom/kabusapi#839]
いつもお世話になっております。
SendOrderで指値注文した後、一定間隔でOrdersを実行し、約定したかどうかを監視するようにしています。
結果をPLAINログで見ると、SendOrderを発行した時刻と、OrdersのDetailsのExecutionDayの時刻との差は、ほとんどの注文では200msec以内なのですが、稀に600msec程度かかる事があります。　
後者の場合、KABUステのT&Sを見る限りでは、指値に対して約定すべき値動きや出来高は十分あるのですが、約定が遅れたのは、どのような原因が考えられるでしょうか？

**Comment by yasuyuki-nakazawa** [comment]
おっしゃっている差というのは、注文を出したタイミングと約定までに600ｍｓがあるがそれはどんなことがあるのかというご質問という理解でよろしいでしょうか

**Comment by zmp365** [comment]
はい。　ログを見ると、SendOrder発行から200msec程度で約定した場合でも、600msec程度で約定した場合でも、どちらでも、SendOrderを発行し、そのResultが返ってくるまで、大体、100～150msec程度かかっています。
SendOrderのResultは同じようなタイミングで返ってくるのに、200msec程度で約定する場合と600msec程度かかる場合があるのです。　
ここで知りたい事は、SendOrderのResultが返ってきた事の意味は、「御社システムが注文を認識した」という意味でしょうか？
或いは、「東証(ArrowHead?）が注文を認識した」という意味でしょうか？
仮に前者の場合、御社システムから東証に注文を引き継ぐ為の所要時間にばらつきがあるという可能性はないでしょうか？
